app/src/main/java/org/kiwix/kiwixmobile/history/HistoryAdapter.java

```diff
@@ -60,7 +63,9 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
       item.itemView.setOnLongClickListener(v ->
           itemClickListener.onItemLongClick(item.favicon, history));
     } else {
-      ((Category) holder).date.setText(historyList.get(position + 1).getDate());
+      SimpleDateFormat format = new SimpleDateFormat("d MMM yyyy", Locale.US);
```

Comment:
Firstly, the date must be formatted using the current locale. Secondly, you should format the date before inserting it in the DB so you don't have to format it after retrieving it from the DB. https://github.com/kiwix/kiwix-android/blob/fe668abde73159961f4468d72966ba6871c7bde0/app/src/main/java/org/kiwix/kiwixmobile/data/local/dao/HistoryDao.java#L29

Reply:
@abdulwd I have done the changes. Please review this.